file_snapshot.go

```diff
@@ -240,7 +240,11 @@ func (f *FileSnapshotStore) getSnapshots() ([]*fileSnapshotMeta, error) {
 		// Ignore any temporary snapshots
 		dirName := snap.Name()
 		if strings.HasSuffix(dirName, tmpSuffix) {
-			f.logger.Printf("[WARN] snapshot: Found temporary snapshot: %v", dirName)
+			f.logger.Printf("[WARN] snapshot: Found temporary snapshot: %v, deleting..", dirName)
```

Comment:
How about adding this logic to ReapSnapshots (https://github.com/hashicorp/raft/blob/8b2b46d7a9e2c2d95d2262afa983e048a2c45ca3/file_snapshot.go#L346-L346) which gets called from snapshot.go via sink.Close. I was primarily concerned with fixing the file system snapshots (other implementations of `SnapshotSink` are inmemory and one that discards it all)